services/manage/app/templates/components/form-elements/toggle-button.hbs

```diff
@@ -0,0 +1,13 @@
+{{#x-toggle
+  size="medium"
+  theme="light"
+  value=value
+  onLabel=onLabel
+  offLabel=offLabel
+  showLabels=true
+  onToggle=(action (mut value)) as |toggle|
+}}
+  {{toggle.offLabel}}
```

Comment:
The dual-label isn't necessary here. A single "Active" label is fine. Similar to NativeX:
![image](https://user-images.githubusercontent.com/3289485/60050726-edc6cb80-9696-11e9-8ddc-8c2604ce61b0.png)